packages/notification-center/src/api/api.service.ts

```diff
@@ -25,8 +25,8 @@ export class ApiService {
     return await this.httpClient.post(`/widgets/messages/${messageId}/seen`, {});
   }
 
-  async getNotificationsList(page: number): Promise<IMessage[]> {
-    return await this.httpClient.get(`/widgets/notifications/feed?page=${page}`);
+  async getNotificationsList(page: number, feedId: string): Promise<IMessage[]> {
+    return await this.httpClient.get(`/widgets/notifications/feed?page=${page}&feedId=${feedId}`);
```

Comment:
So here you may send undefined feedId, it takes me back to the question of the response with no feedId in the usecase https://github.com/novuhq/novu/pull/767/files#r913831986